src/App.tsx

```diff
@@ -2,108 +2,41 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
+import { User } from './types/User';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
 
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
 
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">
-            doloremque illum aliquid sunt
-          </h3>
+  return foundUser || null;
+}
 
-          <p>
-            {' Posted by  '}
+function getComments(postId: number): Comment[] | null {
+  let foundComments = null;
 
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+  if (commentsFromServer.some(comment => comment.postId === postId)) {
+    foundComments = commentsFromServer.filter(comment => comment.postId === postId)
```

Comment:
```suggestion
    foundComments = commentsFromServer
      .filter(comment => comment.postId === postId)
```
do not ignore linter)
<img width="541" alt="Screenshot 2022-10-04 at 11 02 51" src="https://user-images.githubusercontent.com/94289038/193766856-b1c2cc76-ca2e-4d5b-8800-c6ec7a3768c9.png">

Reply:
and no need for this check `commentsFromServer.some(comment => comment.postId === postId` and `let foundComments = null;`
lets just filter and if there will be no match - we get an empty array